theme-base/src/main/assets/themes/mapsforgeV3/base/theme.xml

```diff
@@ -2984,6 +2984,9 @@ version 2.4.0
 					<area fill="#ebeade" />
 				</rule>
 			</rule>
+			<rule e="way" k="man_made" v="cutline" zoom_min="15">
+				<line stroke="#b38d5d" stroke-width="0.2dp" stroke-dasharray="5,5" stroke-linecap="butt" />
+			</rule>
```

Comment:
Interesting, I haven't known that such a tag even exists. Anyway, I'd suggest  to use `base_topo` category to display it only on hike bike and winter theme and increase the size of dashed lines
```suggestion
			<rule cat="base_topo" e="way" k="man_made" v="cutline" zoom_min="15">
				<line stroke="#b38d5d" stroke-width="0.3dp" stroke-dasharray="10,8" stroke-linecap="butt" />
			</rule>
```